rrfs-test/validated_yamls/templates/obtype_config/msonet_winds_288.yaml

```diff
@@ -0,0 +1,513 @@
+           where:
+           - variable: QualityMarker/windEastward
+             is_in: 4-15
+           action:
+             name: reject
+
+         # Initial error assignment
+         - filter: Perform Action
+           filter variables:
+           - name: windEastward
+           - name: windNorthward
+           where:
+           - variable: ObsType/windEastward
+             is_in: 288
+           action:
+             name: assign error
+             error function:
+               name: ObsFunction/ObsErrorModelStepwiseLinear
+               options:
+                 xvar:
+                   name: MetaData/pressure
+                 xvals: [110000, 105000, 100000, 95000, 90000, 85000, 80000, 75000, 70000, 65000, 60000, 55000, 50000, 45000, 40000, 35000, 30000, 25000, 20000, 15000, 10000, 7500, 5000, 4000, 3000, 2000, 1000, 500, 400, 300, 200, 100, 0]
+                 errors: [3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0]
+
+         # Error inflation based on pressure check (setupw.f90)
+         - filter: Perform Action
+           filter variables:
+           - name: windEastward
+           where:
+           - variable: ObsType/windEastward
+             is_in: 288
+           action:
+             name: inflate error
+             inflation variable:
+               name: ObsFunction/ObsErrorFactorPressureCheck
+               options:
+                 variable: windEastward
+                 inflation factor: 4.0
+
+         # Error inflation based on pressure check (setupw.f90)
+         - filter: Perform Action
+           filter variables:
+           - name: windNorthward
+           where:
+           - variable: ObsType/windNorthward
+             is_in: 288
+           action:
+             name: inflate error
+             inflation variable:
+               name: ObsFunction/ObsErrorFactorPressureCheck
+               options:
+                 variable: windNorthward
+                 inflation factor: 4.0
+
+         # Bounds Check
+         - filter: Bounds Check
+           apply at iterations: 0,1
+           filter variables:
+           - name: windEastward
+           - name: windNorthward
+           minvalue: -130
+           maxvalue: 130
+           action:
+             name: reject
+
+         # Gross Error Check
+         - filter: Background Check
+           filter variables:
+           - name: windEastward
+           function absolute threshold:
+           - name: ObsFunction/WindsSPDBCheck
+             options:
+               wndtype:    [ 288 ]
+               cgross:     &cgross    [ 5.0 ]
+               error_min:  &error_min [ 1.0 ]
+               error_max:  &error_max [ 5.0 ]
+               variable: windEastward
+           where:
+           - variable: QualityMarker/windEastward
+             is_not_in: 3
+           action:
+             name: reject
```

Comment:
It looks like this function is setting minimum and maximum errors within the gross error check - am I interpreting that right? Could we use something similar for t/q/v embedded within the gross error checks? Instead of using the individual `Perform Action` filters for min/max error

Reply:
Yes, you are interpreting that correct. In the gross error check there is supposed to be a min/max error used in that check. I would prefer it already built into that filter, but it is not. I don't know why.

Reply:
Is it possible to do the same for t/q/p? So that we wouldn't need those extra filters (e.g., L116-144 in `msonet_airTemperature_188.yaml`).

Reply:
Oh sorry, for some reason I was thinking something else. Please disregard my previous statement... Let me clarify.

All of the gross error checks need to have an error_min error_max in the gross error check. You can do this in the t/q/ps gross checks which I do in the following steps (which I've noted in the comments).

1. Create temporary ObsErrorData
2. Set ObsError set "error parameter" if < "max value"
3. Set ObsError set "error parameter" if > "min value"
4. Gross Error Check
5. Re-assign err ObsErrorData <--- TempObsErrorData after gross error check

Reply:
What I was referring to in my initial statement is that I would like the gross error checks to already have the error min/max built in, but it doesn't so I have to do it in this round about way. The error check for wind does have it built in. I hope that was clear.

Reply:
I understand now, those filter actions are needed for t/q/p to define the min and max errors used during the gross error check. Your explanation was helpful. Hopefully they can be set in the non-wind gross error checks sometime in the future.